commands/a_utility/role-selector.js

```diff
@@ -50,69 +52,73 @@ class RoleSelector extends PermissionCommand {
          */
         let transfers = new Collection();
 
-        const cardEmbed = new MessageEmbed().setColor(botGuild.colors.embedColor)
-            .setTitle('Role Selector!')
-            .setDescription('React to the specified emoji to get the role, un-react to remove the role.');
+        let addTransferFeature = new Feature({
+            name: 'Add a Role!',
+            emojiName: newTransferEmoji,
+            description: 'Add a new emoji to this transfer console! Only staff can select this option!',
```

Comment:
@JPGarCar small thing but would it be better to print the name of the staff role instead of a hardcoded "staff" here?

Reply:
Excellent idea! Will make the change.